docs/our_work/p34_hypergraphs.md

```diff
@@ -3,7 +3,7 @@ layout: base
 title: Exploring Hypergraphs as a Technique for Understanding Impact of Co-Morbidities
 permalink: p34_hypergraphs.html
 summary: In collaboration with Swansea University and the SAIL databank, this work focused on the generation of hypergraphs for investigating the individual and joint impact of comorbidities on a patient pathway.  This work will feed into two future projects to continue the creation of directed hypergraphs and then apply graph neural networks to demonstrate the process of extracting useful insights from these data.
-tags: ['HYPERGRAPHS', 'SAIL']
+tags: []
```

Comment:
Why have the tags been removed? Are no relevant ones in the new schema?

Reply:
yep, readding tags will be a separate piece of work